drivers/nrf24l01p_ng/Makefile.dep

```diff
@@ -0,0 +1,10 @@
+USEMODULE += luid
+USEMODULE += netif
+USEMODULE += xtimer
+ifneq (,$(filter gnrc_ipv6,$(USEMODULE)))
+  USEMODULE += gnrc_sixlowpan_default
```

Comment:
I thought you would be using `gnrc_netif_pktq,` somewhere

Reply:
The transceiver has 1 queue of 3 frames for receiving and 1 queue of 3 frames for transmitting. Maybe I could use `gnrc_netif_pktq` somewhere.